packages/ui/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { MouseEventHandler } from 'react';
+import { forwardRef, HTMLChakraProps, Image as ChakraImage } from '@chakra-ui/react';
+
+import blackHorizontalLogo from '../../../images/LogoBlackHorizontal.svg';
+import blackSquareLogo from '../../../images/LogoBlackSquare.svg';
+import whiteHorizontalLogo from '../../../images/LogoWhiteHorizontal.svg';
+import whiteSquareLogo from '../../../images/LogoWhiteSquare.svg';
+
+type Layout = 'horizontal' | 'square';
+
+type LogoColor = 'white' | 'black';
+
+export interface LogoProps extends HTMLChakraProps<'img'> {
+  layout?: Layout;
+  color?: LogoColor;
+  onClick?: MouseEventHandler;
```

Comment:
I think this component shouldn't receive the `onClick` prop. We probably wouldn't use it right now.

Reply:
removed for now - but i thought that clicking on the logo should redirect user to the dashboard